### ISSUE TITLE
Media page

### ISSUE THREAD

**Issue by Wituareard** [joepio/pauseai#87]
Maybe it would be useful to create a page where you can download different versions of the logo, get the brand color and fonts, get the links to Figma and Google Drive etc.

It could also explain what you should and shouldn't do with it (don't pretend to officially represent PauseAI etc)

And because I expect journalists to click on a page with a name that makes sense for this, it should probably also include an email address.

**Comment by joepio** [comment]
Yes we totally need this! Should we also include links to mentions in the press?

**Comment by Wituareard** [comment]
Thank you for creating the page. I would like to add the brand color and fonts. Are the fonts used on the website the same as those used in other materials?

**Comment by joepio** [comment]
Good idea! Yeah they are the same
